### ISSUE TITLE
Unrecognized extension in file

### ISSUE THREAD

**Issue by richstandbrook** [generoi/sage-woocommerce#1]
Everything has been fine in development. Now I've deployed to a live server provisioned by Laravel Forge (PHP 7.3 then upgraded to  7.4) WP site works fine, but woocommerce pages throw this despicably unhelpful error:

```
2020/09/21 16:30:38 [error] 21747#21747: *4 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught InvalidArgumentException: Unrecognized extension in file: home/forge/site/wordpress/wp-content/themes/godlyplay-2020/resources/views/woocommerce/content-single-product. in /home/forge/site/theme/vendor/illuminate/view/Factory.php:291
Stack trace:
#0 /home/forge/site/theme/vendor/illuminate/view/Factory.php(260): Illuminate\View\Factory->getEngineFromPath()
#1 /home/forge/site/theme/vendor/illuminate/view/Factory.php(122): Illuminate\View\Factory->viewInstance()
#2 /home/forge/site/theme/vendor/roots/acorn/src/helpers.php(408): Illuminate\View\Factory->file()
#3 /home/forge/site/theme/vendor/generoi/sage-woocommerce/src/WooCommerce.php(82): Roots\view()
#4 /home/forge/site/wordpress/wp-includes/class-wp-hook.php(289): Genero\Sage\WooCommerce\WooCommerce->template()
#5 /home/forge/site/wordpress/wp-incl" while reading upstream, client: 212.159.71.194, server: site, request: "GET /product/3-day-accredited-training-course/ HTTP/2.0", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "https://site/"
```

It initial complained about the breadcrumb view, so I removed it (as I hadn't changed it at all) now it's just moved on to complain about another view instead. Just not sure where to look with this.

**Comment by oxyc** [comment]
Sorry I missed this report, could you try and log what argument is passed to this function: https://github.com/generoi/sage-woocommerce/blob/master/src/WooCommerce.php#L67

**Comment by jmerilainen** [comment]
I had similar problem when template files were symlinked. Wrapping `$themeTemplate` to `realpath($themeTemplate)` on line 82 seems to resolve it.

**Comment by oxyc** [comment]
> I had similar problem when template files were symlinked. Wrapping $themeTemplate to realpath($themeTemplate) on line 82 seems to resolve it.

Thanks that makes sense, I added a fix for it in https://github.com/generoi/sage-woocommerce/commit/8ba82e3608427451e7ab62fc122e63d1b0d0dcfd and released v0.1.3. I'll keep this issue open a bit longer in case that wasn't the original issue.
